### PR TITLE
feat: authorize admins and mods to post proposals

### PR DESCRIPTION
**Pull request by bonustrack**
This PR allows admins and moderators to post a proposal regardless of their voting power.

### CODE REVIEW

File: src/writer/proposal.ts [hunk 1]
```diff
@@ -57,11 +57,14 @@ export async function verify(body): Promise<any> {
   }
 
   const onlyAuthors = space.filters?.onlyMembers;
-  const authors = (space.members || []).map(address => address.toLowerCase());
-  const isAuthor = authors.includes(body.address.toLowerCase());
-  if (onlyAuthors && !isAuthor) return Promise.reject('only space authors can propose');
+  const members = (space.members || [])
+    .concat(...(space.admins || []), ...(space.moderators || []))
+    .map(member => member.toLowerCase());
```
Comment (ChaituVR) [inline src/writer/proposal.ts:62]: Why both concat and spread? any advantages?

I feel this is much easier to read:
```suggestion
  const members = (space.members || [])
    .concat(space.admins || [])
    .concat(space.moderators || [])
    .map(member => member.toLowerCase());
```

or maybe
```suggestion
  const members = [
    ...(space.members || []),
    ...(space.admins || []),
    ...(space.moderators || [])
  ].map(member => member.toLowerCase());
```

Comment (bonustrack) [reply src/writer/proposal.ts:62]: Ops its a mistake I started with concat then changed to spread without removing concat

**Review by ChaituVR — APPROVED** [review]
utACK

**Comment by samuveth** [comment]
Can this be merged?

**Pull request merged**